config/index.js

```diff
@@ -75,57 +160,125 @@ module.exports = {
     'no-obj-calls': 'error',
     'no-octal': 'error',
     'no-octal-escape': 'error',
+    'no-param-reassign': 'error',
     'no-path-concat': 'error',
+    'no-plusplus': 'error',
+    'no-process-env': 'off',
+    'no-process-exit': 'error',
     'no-proto': 'error',
+    'no-prototype-builtins': 'error',
     'no-redeclare': 'error',
     'no-regex-spaces': 'error',
     'no-restricted-globals': ['error', 'event'],
+    'no-restricted-imports': 'off',
+    'no-restricted-modules': 'off',
+    'no-restricted-properties': 'off',
+    'no-restricted-syntax': 'off',
     'no-return-assign': ['error', 'except-parens'],
+    'no-return-await': 'error',
+    'no-script-url': 'error',
     'no-self-assign': 'error',
     'no-self-compare': 'error',
     'no-sequences': 'error',
+    'no-setter-return': 'error',
+    'no-shadow': 'error',
```

Comment:
Haha, oh, I see. So this is why there are so many lint problems.

Reply:
Surprisingly only 73 of the 2820 new problems (~3%) in the extension (updated numbers) 🤔

Reply:
Huh, not as bad as I thought! That's good. Hopefully.